models/chainlink/bnb/chainlink_bnb_price_feeds.sql

```diff
@@ -1,59 +1,59 @@
-{{ config(
-    alias = alias('price_feeds'),
-    partition_by = ['block_date'],
-    materialized = 'incremental',
-    file_format = 'delta',
-    incremental_strategy = 'merge',
-    unique_key = ['blockchain', 'block_number', 'proxy_address','underlying_token_address'],
+{{
+  config(
+    tags=['dunesql'],
+    alias='price_feeds',
+    partition_by=['block_date'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key=['blockchain', 'block_number', 'proxy_address','underlying_token_address'],
     post_hook='{{ expose_spells(\'["bnb"]\',
                                 "project",
                                 "chainlink",
-                                \'["msilb7","0xroll"]\') }}'
-    )
+                                \'["msilb7","0xroll","linkpool_ryan"]\') }}'
+  )
 }}
 
+{% set incremental_interval = '7' %}
 {% set project_start_date = '2020-08-29' %}
-{% set answer_updated = '0x0559884fd3a460db3073b7fc896cc77986f16e378210ded43186175bf646fc5f' %}
 
-
-SELECT 'bnb'                                        AS blockchain,
+SELECT 'bnb' as blockchain,
        c.block_time,
        c.block_date,
        c.block_number,
        c.feed_name,
        c.oracle_price,
        c.proxy_address,
        c.aggregator_address,
-       COALESCE(o.underlying_token_address, 'n/a')      AS underlying_token_address,
-       c.oracle_price / POWER(10 , o.extra_decimals)    AS underlying_token_price
+       o.underlying_token_address,
+       c.oracle_price / POWER(10, o.extra_decimals) as underlying_token_price
 FROM
 (
-    SELECT l.block_time,
-           DATE_TRUNC('day', l.block_time)              AS block_date,
-           l.block_number,
-	       cfa.feed_name,
-           AVG(
-             conv( --handle for multiple updates in the same block
-                  substring(l.topic2,3,64),16,10)
-                  /POWER(10, cfa.decimals)
-              )                                         AS oracle_price,
-	       cfa.proxy_address,
-           cfa.aggregator_address
-	FROM {{ source('bnb', 'logs') }} l
-	INNER JOIN {{ ref('chainlink_bnb_oracle_addresses') }} cfa ON l.contract_address = cfa.aggregator_address
-	WHERE l.topic1 = '{{answer_updated}}'
+    SELECT
+        l.block_time,
+        cast(date_trunc('day', l.block_time) as date) as block_date, 
```

Comment:
if `block_date` is used solely for the partition, we are looking to universally change this to a higher level of granularity, `block_month`.

can you update to be `'month'` instead, then rename column and partition in config to `block_month`?

if you use `block_date` for more than the partition, then you can simply add new column instead. in fact, that may be safer if downstream queries on the app depend on this column for whatever reason.

Reply:
this would also apply universally, only commenting here